### ISSUE TITLE
Super quantum catch

### ISSUE THREAD

**Issue by thomased** [rmaia/pavo#70]
Would make `vismodel()` totally unwieldy, but could perhaps think about it via some auxiliary function for the keen/data rich. 

<img width="421" alt="blah" src="https://user-images.githubusercontent.com/6053453/32477579-74e5017e-c3d3-11e7-9e25-e854c0db1f2f.png">

**Comment by Bisaloo** [comment]
Can you add a link to the source alongside the screenshot for future reference please?

**Comment by thomased** [comment]
Yeah, was just trying to remember — [http://dx.doi.org/10.1098/rstb.2016.0338](http://dx.doi.org/10.1098/rstb.2016.0338)

I'm not committed to the idea, it just came up in a review and brought it to mind, so I'll give it a little more thought.
